spock-specs/src/test/resources/org/spockframework/smoke/Interceptors__interceptors_are_called_in_the_correct_order.txt

```diff
@@ -0,0 +1,404 @@
+shared initializer start (SubSpec)
+  shared initializer method start (SuperSpec.$spock_initializeSharedFields())
+  shared initializer method end (SuperSpec.$spock_initializeSharedFields())
+  shared initializer method start (SubSpec.$spock_initializeSharedFields())
+  shared initializer method end (SubSpec.$spock_initializeSharedFields())
+shared initializer end (SubSpec)
```

Comment:
What was your motivation to add the interceptor to each initializer method, but only one for the shared initializer of the SubSpec? Same for the other ones as well, ie., why aren't we putting `specInfo.specsBottomToTop*.` on all non-feature interceptors?

Reply:
🤦‍♂️
You should have asked this question before merging #1667, because this simple question makes #1667 incomplete.
I guess I thought the ones for the super spec will not trigger in this case.
Indeed the "specification" one is not called, but "shared initializer", "setup spec", "initializer", "setup", "cleanup", and "cleanup spec" are called nested within the one of the subspec.